packages/app/src/explorer/utils.ts

```diff
@@ -26,3 +26,28 @@ export function needsNxBadge(
 
   return false;
 }
+
+export function getActiveIndex(
+  buttonList: HTMLButtonElement[],
+  element: Element
+): number | undefined {
+  const { activeElement } = document;
+
+  if (
+    activeElement instanceof HTMLButtonElement &&
+    element.contains(activeElement)
+  ) {
+    return buttonList.lastIndexOf(activeElement);
```

Comment:
I think `indexOf` would suffice here, since we don't expect the same button instance to appear multiple times (not technically possible). Same below :point_down: